docs/event-dispatcher.md

```diff
@@ -105,6 +152,76 @@ Example:
 }
 ```
 
+#### Example json values for burnchain operations 
+- TransferStx 
+```json
+{
+  "TransferStx": {
+    "block_height": 10,
+    "burn_header_hash": "1410131010105914101010101013704010101010221010101010101010101010",
+    "memo": "0x000102030405",
+    "recipient": {
+      "address": "SP24ZBZ8ZE6F48JE9G3F3HRTG9FK7E2H6K2QZ3Q1K",
+      "address_hash_bytes": "0x89f5fd1f719e4449c980de38e3504be6770a2698",
+      "address_version": 22
+    },
+    "sender": {
+      "address": "ST2QKZ4FKHAH1NQKYKYAYZPY440FEPK7GZ1R5HBP2",
+      "address_hash_bytes": "0xaf3f91f38aa21ade7e9f95efdbc4201eeb4cf0f8",
+      "address_version": 26
+    },
+    "transfered_ustx": 10,
+    "txid": "85aa2106186723f3c4f1d8bb58e3a02746ca9be1be9f4be0c6557079e1f660e6",
+    "vtxindex": 10
+  }
+}
+```
+
+- StackStx
+```json
+{
+  "StackStx": {
+    "block_height": 10,
+    "burn_header_hash": "1010101010101010101010101010101010101010101010101010101010101010",
+    "num_cycles": 10,
+    "reward_addr": "16Jswqk47s9PUcyCc88MMVwzgvHPvtEpf",
```

Comment:
Does this need the new JSON format?

Reply:
Meant to say, I see that this format is different from `DelegateStx`.  Is this intentional?

Reply:
It's different because the actual field of reward_addr has different types for StackStx (`PoxAddress`) and DelegateStx (`Option<(u32, PoxAddress)>`